tasks/init-script.yml

```diff
@@ -19,4 +19,4 @@
   when: ansible_os_family == "Debian"
 
 - name: Ensure solr is started and enabled on boot.
-  service: name=solr state=started enabled=yes
+  service: name=solr state=started enabled=yes runlevel=2
```

Comment:
Can you please leave this change out of this PR? I'm happy to merge the solr_host change immediately, but would rather discuss a runlevel default change separately :)